src/blocks/Pricing/index.tsx

```diff
@@ -87,17 +98,17 @@ export const Pricing: React.FC<Props> = ({ pricingFields }) => {
                         </CollapsibleToggler>
                       </Collapsible>
                     </div>
-                  </Cell>
+                  </div>
                 )
               })}
               {disclaimer && (
-                <Cell>
+                <div className={[].filter(Boolean).join(' ')}>
```

Comment:
@paulpopus Why did you leave out columns here?

Reply:
@PatrikKozak this component is getting redesigned completely, the disclaimer isn't current in use on prod either so i'm inclined to leave it like this for now